documentation/rdme.mdx

```diff
@@ -0,0 +1,331 @@
+
+Follow these steps to manually configure a GitHub Actions Workflow. This will require some familiarity with the [workflow syntax](https://docs.github.com/actions/using-workflows/workflow-syntax-for-github-actions)
+
+1. Create a new [GitHub Actions workflow file](https://docs.github.com/actions/using-workflows/workflow-syntax-for-github-actions) in the `.github/workflows` directory of your repository (or use an existing workflow file)
+
+1. Configure the [`on`](https://docs.github.com/actions/using-workflows/workflow-syntax-for-github-actions#on) property, which determines what triggers the execution of the workflow.
+
+1. Add the following two [steps](https://docs.github.com/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idsteps) to your workflow:
+
+```yml
+# Required in order for the GitHub Action to access your repo's contents
+- uses: actions/checkout@v4
+
+# Runs the `rdme` command on your repo's contents
+- uses: readmeio/rdme@RDME_VERSION
+  with:
+    rdme: [your command here]
+```
+
+The command syntax in GitHub Actions is functionally equivalent to the CLI. For example, take the following CLI command:
+
+```sh
+rdme openapi validate [url-or-local-path-to-file]
+```
+
+To execute this command via GitHub Actions, the step would look like this:
+
+```yml
+- uses: readmeio/rdme@RDME_VERSION
+  with:
+    rdme: openapi validate [url-or-local-path-to-file]
+```
+
+The following section has links to full GitHub Actions workflow file examples.
+
+> 👍 Did you know?
+>
+> Not to get too meta on you, but... the page that you're currently reading is actually being synced from the `rdme` GitHub repository via the `rdme` GitHub Action 🤯
+>
+> Here are the relevant files on GitHub:
+>
+> - [The Markdown source file for the page you're reading](https://github.com/readmeio/rdme/blob/main/documentation/rdme.md) 📜
+> - [The GitHub Actions workflow file that syncs the Markdown to docs.readme.com](https://github.com/readmeio/rdme/blob/main/.github/workflows/docs.yml) 🔄
+> - And finally... [the workflow run results](https://github.com/readmeio/rdme/actions/workflows/docs.yml) ✅
+
+### GitHub Actions Examples
+
+> 🚧 ReadMe Refactored Guidance
+>
+> This guidance is only applicable for projects that have been migrated to [ReadMe Refactored](https://docs.readme.com/main/docs/welcome-to-readme-refactored). The Refactored project architecture requires `rdme@10`, while the legacy project architecture requires `rdme@9`.
+>
+> If you are still on our legacy project architecture, please see [our documentation for `rdme@9`](https://docs.readme.com/docs/rdme-legacy).
+
+Want to start syncing? We have several example workflow files available:
+
+// TODO: create new guides for GHA with `rdme@10` and update these links
+
+- [Syncing an OpenAPI definition](https://docs.readme.com/docs/github-actions-openapi-example)
+- [Syncing a directory of Markdown files](https://docs.readme.com/docs/github-actions-docs-example)
```

Comment:
Do we need to create new guides for v10?

Reply:
I think besides the "Automagical" file generation, the rest of it is still relevant, right?